[15.0][IMP] l10n_es_factuare: mejoras en el readme

JanQubiQ opened the pull request:
Propuesta de mejora del Readme haciéndolo más explicativo en los apartados de configuración y uso

sergio-teruel committed:
[FIX] l10n_es_aeat: Don't error on manually created export lines

enriquemartin committed:
[FIX] l10n_es_ticketbai_api_batuz - normalizar cálculo de ejercicio

ramiadavid committed:
[FIX] l10n_es_aeat_mod347: Formato informe PDF

Commit:
[15.0][IMP] l10n_es_factuare: mejoras en el readme

fix

Commit:
[15.0][IMP] l10n_es_factuare: mejoras en el readme

Commit:
[15.0][IMP] l10n_es_factuare: mejoras en el readme

oihane committed:
[FIX] l10n_es_ticketbai: ignore section lines

OCA-git-bot committed:
Merge PR #2744 into 15.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_ticketbai 15.0.1.4.4

OCA-git-bot committed:
[UPD] addons table in README.md

etobella commented on l10n_es_facturae/__manifest__.py:
```diff
@@ -8,7 +8,7 @@
 
 {
     "name": "Creación de Facturae",
-    "version": "15.0.1.4.0",
```
No cambios la versión. Esto lo hace el ocabot

JanQubiQ replied:
Perdona, ya está resuelto!

Commit:
fix

etobella commented:
Y si puedes junta los commits con git rebase. En general si quieres editar el último commit puedes usar git commit --amend

OCA-git-bot committed:
Merge PR #2747 into 15.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_ticketbai_api_batuz 15.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2750 into 15.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_aeat 15.0.2.4.0

OCA-git-bot committed:
[UPD] addons table in README.md

ao-landoo committed:
[FIX] l10n_es_ticketbai: Display TBAI ID and QR always on same page

ferran-S73 committed:
[FIX] l10n_es_aeat_mod347: cáclulo importe entregas

ao-landoo committed:
[IMP] l10n_es_aeat_mod390: Soporte IVA 5%

Shide committed:
[IMP] l10n_es_aeat_sii_oca: Includes new taxes 0%, 0.62% and 5%

ramiadavid committed:
[FIX] l10n_es_aeat: Remove old migration script

ramiadavid committed:
[FIX] l10n_es_aeat_mod347: Remove old migration script

OCA-git-bot committed:
Merge PR #2764 into 15.0

Signed-off-by pedrobaeza

pedrobaeza committed:
[IMP] l10n_es_aeat_mod303: Convert 303 activity codes data to CSV

For faster load and maintenance

pedrobaeza committed:
[FIX] l10n_es_aeat_mod303: Adapt to 2022 exonerated reqs

- Activity codes are now 3 letters long instead of integers.
- New activity codes for this year, and shared for 4T and 12.
- Implemented new date_start and date_end on activity codes, period_type
  not required and code as char, along with an extended domain on 303
  report for accomodating to the new specs.
- Fix BOE format to real specs.

OCA-git-bot committed:
Merge PR #2753 into 15.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_ticketbai 15.0.1.4.5

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2757 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat_mod347.pot

OCA-git-bot committed:
l10n_es_aeat_mod347 15.0.1.2.0

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_aeat_mod347/

OCA-git-bot committed:
Merge PR #2733 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat_mod347.pot

OCA-git-bot committed:
l10n_es_aeat_mod347 15.0.1.3.0

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_aeat_mod347/

OCA-git-bot committed:
Merge PR #2761 into 15.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_aeat_sii_oca 15.0.2.8.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2772 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat_mod303.pot

OCA-git-bot committed:
l10n_es_aeat_mod303 15.0.2.4.0

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_aeat_mod303
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_aeat_mod303/

eantones committed:
[FIX] l10n_es_aeat_mod349: Operation key not taken into account in refunds

Steps to reproduce the problem:

- Create an intra-community invoice on 2T with one service and one
  product, having 2 separate operation keys.
- Refund the invoice on 3T.
- Create a 349 declaration for 3T.
- On refund of others periods page, both refunded amounts are charged to
  the first operation key.

pedrobaeza committed:
[FIX] l10n_es_aeat_mod390: Adapt to 2022 BOE format + changes

- New activity codes
- New fields for 5%
- New BOE format

TODO: New 2023 taxes (0% food national, intra-community and Req)

rafaelbn commented:
@JanQubiQ gracias! Por favor puedes mirar el último comentario de Enric?

pedrobaeza committed:
[FIX] l10n_es_aeat_mod303: Domain for other activity codes

They were not adjusted to new domain and thus, you can't select the
proper activity codes for this year.

TT41421

pedrobaeza committed:
[FIX] Update copier template

To avoid the isort problem (PyCQA/isort#2077)

OCA-git-bot committed:
Merge PR #2783 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat_mod390.pot

OCA-git-bot committed:
l10n_es_aeat_mod390 15.0.2.0.0

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_aeat_mod390
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_aeat_mod390/

OCA-git-bot committed:
Merge PR #2792 into 15.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_aeat_mod303 15.0.2.5.0

OCA-git-bot committed:
[UPD] addons table in README.md

cesarlr committed:
[FIX] l10n_es_mod390: Set main_activity_code as required only for calculated state

OCA-git-bot committed:
Merge PR #2800 into 15.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_aeat_mod390 15.0.2.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2803 into 15.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
l10n_es_aeat_mod349 15.0.1.3.1

OCA-git-bot committed:
[UPD] addons table in README.md

polqubiq committed:
Merge branch '15.0-imp-l10n_es_facturae-imp-readme' of github.com:QubiQ/l10n-spain into 15.0-imp-l10n_es_facturae-imp-readme

polqubiq committed:
Merge branch '15.0-imp-l10n_es_facturae-imp-readme' of github.com:QubiQ/l10n-spain into 15.0-imp-l10n_es_facturae-imp-readme
